code/obj/item/rcd.dm

```diff
@@ -415,6 +418,72 @@ Broken RCD + Effects
 						W.attach_light_fixture_parts(user, LB, TRUE)
 						log_construction(user, "built a light fixture to a wall ([W])")
 
+ // Express limb surgery with an RCD
+	attack(mob/living/carbon/human/M as mob, mob/living/carbon/user as mob)
+
+		var/obj/item/parts/surgery_target = null
+		var/is_missing = 0
```

Comment:
I'm confused about the usage of this- we return early if they don't have the limb/organ we're trying to RCD off, but later on it checks those again and uses `is_missing`. Unless I'm missing something I don't see a way for the code on line 450 to be reached, since we would have returned early if they were missing the limb.

Reply:
It may not be the best way to do it, but the is_missing is only used when clumsy kicks in. We did check earlier if the target's limb was there, but we have to check again later if the user's limb is there.

Reply:
Ahh, I see. would appreciate renaming it to `user_limb_is_missing` or something so future people don't make the same mistake I did

Reply:
Done so